docs/en_us/internal/testing.rst

```diff
@@ -94,6 +94,22 @@ Test Locations
    -  Bok Choy Accessibility Tests: located under ``common/test/acceptance/accessibility``
    -  Bok Choy PageObjects: located under ``common/test/acceptance/pages``
 
+-  Content type examples:
+
+   The Accessibility Mega Course
+   `https://github.com/cptvitamin/demo-test-course
+   <https://github.com/cptvitamin/demo-test-course>`_ provides
+   a comprehensive collection of all edX content types. The course exercises
+   all possible Studio content creation and is a valuable testing resource.
+   It is available by default in sandboxes.
+
+   The Demo Performance Course
+   `https://github.com/edx/demo-performance-course
+   <https://github.com/edx/demo-performance-course>`_ is a demonstration
+   course you can import into edX Studio. Use it to evaluate performance
+   implications of individual components.
+
```

Comment:
This course is the one that is available for free from sandboxes; however, I don't think we should mention sandboxes on this document anyway. That is something only available to folks that work here and have access to the various networks.

Cutting to the chase: this particular section LGTM.